fastlane_core/lib/fastlane_core/fastlane_pty.rb

```diff
@@ -31,6 +40,11 @@ def self.spawn(command)
         command_stdout.close
         p.value.exitstatus
       end
+    rescue StandardError => e
```

Comment:
`StandardError` means all Exceptions that can happen?

Reply:
Yup yup! This will catch everything that could happen

Reply:
TIL Mother of all errors, so to speak.